extensions/positron-python/src/client/common/utils/localize.ts

```diff
@@ -266,7 +266,7 @@ export namespace InterpreterQuickPickList {
 
 export namespace OutputChannelNames {
     export const languageServer = l10n.t('Python Language Server');
-    export const python = l10n.t('Python');
+    export const python = l10n.t('Python Language Pack');
```

Comment:
I believe we need our [overlays](https://connect.posit.it/positron-wiki/overlay-strategy.html) in this file, to help with upstream merges of this extension.

Reply:
Oh, you are right! Thank you for catching that.